FlowCrypt/Models/Realm Models/KeypairRealmObject.swift

```diff
@@ -83,6 +83,7 @@ extension KeypairRealmObject {
     static func createPrimaryKey(primaryFingerprint: String, email: String) -> String {
         var hash = SHA256()
         hash.update(data: primaryFingerprint.data())
+        hash.update(data: Data(count: 1))
```

Comment:
Is this a null character? At least should be explicit. Let's do `",".data()`

Reply:
Yes, one zero byte.

Reply:
Are you sure that `,` could not be injected with `primaryFingerprint` or `email`?

Reply:
You're right. I'm more sure that a null byte wouldn't get through than a comma. I've reverted it back to what you did.